### PR TITLE
Upgrade UPP with netcdf_parallel library 4.7.4, hdf5_parallel 1.10.6 …

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
…and w3emc 2.4.0.

### CODE REVIEW

File: modulefiles/post/v8.0.0-hera [hunk 1]
```diff
@@ -36,13 +36,16 @@ module load g2/3.1.1
...
-module load w3emc/2.3.1
+#module load w3emc/2.3.1
 module load w3nco/2.0.7
 module load crtm/2.2.5
 #module load netcdf/3.6.3
-module load netcdf/4.7.0
+#module load netcdf/4.7.0
 module load g2tmpl/1.6.0
 module load wrfio/1.1.1
+module load hdf5_parallel/1.10.6
```
Comment (yangfanglin) [inline modulefiles/post/v8.0.0-hera:46]: add "module use /scratch2/NCEPDEV/nwprod/NCEPLIBS/modulefiles" to use these new modules

**Review by yangfanglin — COMMENTED** [review]
Add "module use /scratch2/NCEPDEV/nwprod/NCEPLIBS/modulefiles" to load the new modules on Hera

File: modulefiles/post/v8.0.0-hera (hunk 1 above)
Comment (WenMeng-NOAA) [reply modulefiles/post/v8.0.0-hera:46]: This module path is already added at the line 24 in v8.0.0-hera.

Comment (yangfanglin) [reply modulefiles/post/v8.0.0-hera:46]: I see. The view window had that cut off.

**Review by yangfanglin — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
The regression tests were conducted on Dell and Hera. No changed results were found.

**Pull request merged**